src/components/TheTypography.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Poppins</title>
+</head>
+    <body>
+
+    <div class="container">
+      <p class="poppins">Poppins</p>
+    </div> 
+
+    <!-- Heading 1 -->
+
+    <div class="container"><hr>
+      <div class="row align-items-start">
+        <div class=" col ">
+           <p class="Poppins-H1-SemiBold"> Display 01 Semi Bold</p>
+        </div>
+        <div class="col">
+           <p class="Poppins-H1-Bold"> Display 01 Bold </p>
+        </div>
+        <div class="col">
+          <p class="Poppins-H1-Black">Display 01 Black </p>
```

Comment:
bakit puro <p> tag gamit mo sa mga heading since may identifier ka na H1 n class
pa bago din ng naming ng class di tayo ang cacamel case sa mga class name